LocalSettings.php

```diff
@@ -477,6 +481,8 @@
 			'editor' => true,
 			'sysop' => true,
 		),
+		'+wisdomsandboxwiki' => array(
+			'anon' => true,)
```

Comment:
the ) needs to be on a new line lined up with the start of line 484, and it needs a , after it